### PR TITLE
Matter power spectrum with camb backend

### PR DESCRIPTION
**Pull request by marcelo-alvarez**
Modifies the mockgen CLI to set the matter power spectrum using the CAMB backend of xgcosmo.

### CODE REVIEW

File: mockgen/sky.py [hunk 1]
```diff
@@ -23,6 +20,8 @@ def __init__(self, **kwargs):
         self.Nside    = kwargs.get(   'Nside',mgd.Nside)
         self.gpu      = kwargs.get(     'gpu',mgd.gpu)
         self.mpi      = kwargs.get(     'mpi',mgd.mpi)
+        self.h        = kwargs.get(       'h',mgd.h)
```
Comment (1cosmologist) [inline mockgen/sky.py:23]: Instead of passing individual cosmology parameters can we pass a dictionary of cosmology params?

**Review by 1cosmologist — CHANGES_REQUESTED** [review]
Instead of h and omega_m take a dictionary of cosmology parameters as input.

File: mockgen/sky.py (hunk 1 above)
Comment (marcelo-alvarez) [reply mockgen/sky.py:23]: Thanks for the suggestion @1cosmologist. I agree that it is a useful enhancement we should implement and I suggest we do it outside of this PR -- I have created #3 to track this.

Comment (1cosmologist) [reply mockgen/sky.py:23]: Ok, that sounds good.

**Review by 1cosmologist — APPROVED** [review]

**Comment by 1cosmologist** [comment]
We are ready to merge!

**Pull request merged**